tests/unit/test_configuration.py

```diff
@@ -0,0 +1,10 @@
+"""
+
+"""
+from src.config import PLUGIN_MANAGER
+
+
+def test_rehash(configuration_fx):
+    PLUGIN_MANAGER.hook.rehash_handler(data=configuration_fx)
+def test_validate(configuration_fx):
```

Comment:
[Style] Newline between method definitions.

Reply:
this file wasn't supposed to be commited yet... sigh.